### PR TITLE
Hot fix for beta release

### PR DESCRIPTION
**Pull request by Ismael034**
Add sharedPreference of language on first start and added missing stings in strings.xml

### CODE REVIEW

File: ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/ActivityLauncherApp.java [hunk 1]
```diff
@@ -21,5 +21,8 @@ public void onCreate() {
         if (!prefs.contains("hide_private_activities")) {
             prefs.edit().putBoolean("hide_private_activities", true).apply();
         }
+        if (!prefs.contains("locale")) {
```
Comment (butzist) [inline ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/ActivityLauncherApp.java:24]: Add space before block

File: ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/ActivityLauncherApp.java [hunk 2]
```diff
@@ -21,5 +21,8 @@ public void onCreate() {
         if (!prefs.contains("hide_private_activities")) {
             prefs.edit().putBoolean("hide_private_activities", true).apply();
```
Comment (butzist) [inline ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/ActivityLauncherApp.java:22]: False, by default

File: ActivityLauncherApp/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -50,4 +50,6 @@
     <string name="theme_default">System Default</string>
     <string name="theme_light">Light Theme</string>
     <string name="theme_dark">Dark Theme</string>
+    <string name="private_activities_summary">Hide private (unexported and disabled) activites, that cannot be started without root access</string>
```
Comment (butzist) [inline ActivityLauncherApp/src/main/res/values/strings.xml:53]: Please place below related setting and rename to settings_description_hide_private

Comment (butzist) [reply ActivityLauncherApp/src/main/res/values/strings.xml:53]: Or summary

File: ActivityLauncherApp/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -50,4 +50,6 @@
...
+    <string name="root_mode_summary">Allow launching private activities by using root privileges</string>
```
Comment (butzist) [inline ActivityLauncherApp/src/main/res/values/strings.xml:54]: Same as above, name should refer to the setting, that it's related to

**Review by butzist — COMMENTED** [review]
The names in the activity list remain in System language independent of this setting

**Comment by Ismael034** [comment]
> The names in the activity list remain in System language independent of this setting
Interesting. I didnt notice that. I am not sure If we can fix that as the names comes from `getApplicationLabel` from PackageManager which depends on the System language. I dont know if we could trick the system into returning a name in another language

**Pull request merged**